devicetypes/smartthings/arrival-sensor-ha.src/arrival-sensor-ha.groovy

```diff
@@ -79,8 +101,27 @@ def beep() {
 }
 
 def parse(String description) {
+    log.debug "description: $description"
     state.lastCheckin = now()
-    handlePresenceEvent(true)
+    if (isVision()) {
+        if (description?.startsWith("catchall:")) {
+            log.debug zigbee.parseDescriptionAsMap(description)
```

Comment:
just log `descMap` after you've parsed it instead of parsing it twice

Reply:
OK! I will modify the code.